src/content/docs/features/window-customization.mdx

```diff
@@ -2,13 +2,207 @@
+| `window:allow-close`                    | Enables the close command without any pre-configured scope.                    |
+| `window:allow-minimize`                 | Enables the minimize command without any pre-configured scope.                 |
+| `window:allow-start-dragging`           | Enables the start_dragging command without any pre-configured scope.           |
+| `window:allow-toggle-maximize`          | Enables the toggle_maximize command without any pre-configured scope.          |
+| `window:allow-internal-toggle-maximize` | Enables the internal_toggle_maximize command without any pre-configured scope. |
+
+#### CSS
+
+You'll need to add some CSS for the titlebar to keep it at the top of the screen and style the buttons:
+
+```css
+.titlebar {
+	height: 30px;
+	background: #329ea3;
+	user-select: none;
+	display: flex;
+	justify-content: flex-end;
+	position: fixed;
+	top: 0;
+	left: 0;
+	right: 0;
+}
+.titlebar-button {
+	display: inline-flex;
+	justify-content: center;
+	align-items: center;
+	width: 30px;
+	height: 30px;
+	user-select: none;
+	-webkit-user-select: none;
+}
+.titlebar-button:hover {
+	background: #5bbec3;
+}
+```
+
+#### HTML
+
+Now, you'll need to add the HTML for the titlebar. Put this at the top of your `<body>` tag:
+
+```html
+<div data-tauri-drag-region class="titlebar">
+	<div class="titlebar-button" id="titlebar-minimize">
+		<img
+			src="https://api.iconify.design/mdi:window-minimize.svg"
+			alt="minimize"
+		/>
+	</div>
+	<div class="titlebar-button" id="titlebar-maximize">
+		<img
+			src="https://api.iconify.design/mdi:window-maximize.svg"
+			alt="maximize"
+		/>
+	</div>
+	<div class="titlebar-button" id="titlebar-close">
+		<img src="https://api.iconify.design/mdi:close.svg" alt="close" />
+	</div>
+</div>
+```
+
+Note that you may need to move the rest of your content down so that the titlebar doesn't cover it.
+
+#### JS
+
+Finally, you'll need to make the buttons work:
+
+```js
+import { Window } from '@tauri-apps/api/window';
+
+const appWindow = new Window('main');
+
+document
+	.getElementById('titlebar-minimize')
+	?.addEventListener('click', () => appWindow.minimize());
+document
+	.getElementById('titlebar-maximize')
+	?.addEventListener('click', () => appWindow.toggleMaximize());
+document
+	.getElementById('titlebar-close')
+	?.addEventListener('click', () => appWindow.close());
+```
+
+### (macOS) Transparent Titlebar with Custom Window Background Color
+
+For macOS, using a custom titlebar will also lose some features provided by the system, such as [moving or aligning the window](https://support.apple.com/guide/mac-help/work-with-app-windows-mchlp2469/mac). Another approach to customizing the titlebar but keeping native functions could be making the titlebar transparent and setting the window background color.
+
+We are going to create the main window and change the window background color from the Rust side, so here we remove the main window from the `tauri.conf.json` file:
```

Comment:
```suggestion
We are going to create the main window and change its background color from the Rust side.

Remove the main window from the `tauri.conf.json` file:
```
It's not exactly this markup, but kinda of, will have to fix it in markdown to make sure it's ok